packages/edit-site/src/components/revisions/index.js

```diff
@@ -88,6 +87,7 @@ function Revisions( { userConfig, blocks } ) {
 						settings={ settings }
 					>
 						<BlockList renderAppender={ false } />
+						<EditorStyles styles={ editorStyles } />
```

Comment:
So now, the overrides store reflects the current state as the block list as been rendered and `useBlockProps` run.

However, now I need to merge the revisions style data with existing variations+clientId rules.

Working on that now...